FindBoringSSL.cmake

```
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2021 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+#
+
+include("GenericFindDependency")
+GenericFindDependency(
+  TARGET ssl
+  SOURCE_DIR boringssl
```

```suggestion
  ADDITIONAL_TARGETS crypto
  SOURCE_DIR boringssl
```

Lets add `ADDITIONAL_TARGETS` so we also make sure that the crypto library was found and configured correctly.